### PR TITLE
nightly builds + coverity scan

### LINKED ISSUE #4190: move netdata from firehol/netdata to netdata/netdata
Response from github support, about moving firehol/netdata to netdata/netdata.

---

If you transfer the repository, we'll set up redirects from the old URL to the new URL. You can read more about what happens when you transfer a repository here: https://help.github.com/articles/about-repository-transfers/

Note that these redirects will only remain in place as long as you don't create a new repository in the old location with the same name.

You can find the steps to start the transfer here: https://help.github.com/articles/transferring-a-repository-owned-by-your-organization/

I'll answer your individual questions below:

> the whole repo, including issues, wiki, etc will be moved?

Yes, all that content will be moved.

> what will happen to URLs pointing to the old firehol/netdata path?

Anyone visiting https://github.com/firehol/netdata will be redirected to https://github.com/netdata/netdata.

> will users that have checked out firehol/netdata pull updates from netdata/netdata?

Yes, we also redirect Git remote URLs, so these folks will also be pulling from the new repository.

We still recommend that folks do update their bookmarks and remote URLs to point to the new repository location to save any confusion. You may want to add a note on your repository's README to notify your users.

### PR DESCRIPTION
**Pull request by paulfantom**
- Stabilizes number of builds
- Enables automatic coverity scans and closes #4190

Two things needs to be done via travis GUI:
- enable cron based builds
- add `COVERITY_SCAN_TOKEN` variable

### CODE REVIEW

**Comment by ktsaou** [comment]
Covertity scan token added: https://travis-ci.org/netdata/netdata/settings

**Comment by paulfantom** [comment]
Wrong travis :stuck_out_tongue:

**Comment by ktsaou** [comment]
hm... `.com` took it too.

**Comment by paulfantom** [comment]
.com was added manually by me.

File: coverity-scan.sh [hunk 1]
```diff
@@ -1,38 +1,54 @@
...
+	echo "Cannot find 'cov-build' binary in \$PATH."
+	if [ $INSTALL_COVERITY != "" ]; then
+		echo "Installing coverity..."
+		mkdir /tmp/coverity
+        	curl -SL --data "token=${token}&project=netdata%2Fnetdata" https://scan.coverity.com/download/linux64 > /tmp/coverity_tool.tar.gz
+        	tar -x-C /tmp/coverity/ -f /tmp/coverity_tool.tar.gz
+	        sudo mv /tmp/coverity/cov-analysis-linux64-2017.07 /opt/coverity
+        	export PATH=${PATH}:/opt/coverity/bin/
+        else
+		echo "Saving command the full filename of cov-build in .coverity-build"
```
Comment (ktsaou) [inline coverity-scan.sh]: typo. It should say `Save the full filename of the cov-build command in .covertity-build`.

**Comment by paulfantom** [comment]
fixed

**Comment by ktsaou** [comment]
> fixed

It still makes no sense, but let proceed...

**Pull request merged**